src/common/paddle/index.ts

```diff
@@ -240,4 +250,30 @@ export const insertClaimableItem = async (
       `User ${customer.email} already has a claimable subscription`,
     );
   }
+
+  await identifyAnonymousFunnelSubscription({
+    cio,
+    email: customer.email,
+    claimedSub: false,
+  });
+};
+
+export const dropClaimableItem = async (
+  con: DataSource | EntityManager,
```

Comment:
I think we have a common type for this.
```suggestion
  con: ConnectionManager,
```